app/assets/javascripts/govuk_publishing_components/components/skip-link.js

```diff
@@ -0,0 +1,5 @@
+// This component relies on JavaScript from GOV.UK Frontend
+// = require govuk/components/skip-link/skip-link.js
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {}
+window.GOVUK.Modules.GovukSkipLink = window.GOVUKFrontend.SkipLink
```

Comment:
@andysellick 

---

This looks like a new JS file for the skip link component. This needs to be highlighted for deployment to those applications that use this component as they'll need to include it. I don't think it technically counts as a breaking change, but maybe we should highlight it as that to ensure this gets done?

Applications using skip link are `collections-publisher, content-data-admin, content-publisher, datagovuk_find, finder-frontend, government-frontend, govuk-coronavirus-vulnerable-people-form, release, search-admin, signon, static`. Might be worth also checking this setup, I'm not surprised it's in `static` but a little unsure why it's in some of the others e.g. `finder-frontend`.

Reply:
I initially left a note in the PR description about the associated script for this component needing to be imported in `static` – will make sure I'll add this after this change is released.